### DISCUSSION TITLE
Overview of Bicep – Next generation ARM templates [Azure Demo]

### DISCUSSION THREAD

**Discussion by RogalaPiotr** [AzureCommunityPL/group#283] [Wydarzenia]
Zapraszam wszystkich zainteresowanych automatyzacją w Microsoft Azure na spotkanie jednym z twórców Bicep. Nasze spotkanie odbędzie się w języku angielskim i Alex pokaze nam deep dive z Bicep. W czasie spotkania zapraszamy do dyskusji!

Data: 20.05.2021 (Czwartek)
Godzina: 17.00

Platfroma: Teams oraz Youtube

Adres do wydarzenia:
https://www.eventbrite.com/e/overview-of-bicep-next-generation-arm-templates-azure-demo-wrocaw-tickets-153947275445

Link do Youtube zostanie wrzucony w komentarz pod wydarzeniem, parę minut przed rozpoczęciem.

**Comment by RogalaPiotr** [comment]
https://youtu.be/SENIu-mnMaE

**Comment by qqbek24** [comment]
Super, dziękuję :)

czw., 20 maj 2021 o 17:09 RogalaPiotr ***@***.***> napisał(a):
